tests/DependencyInjection/Compiler/ThumbnailCompilerPassTest.php

```diff
@@ -11,10 +11,9 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\DependencyInjection\Tests\Compiler;
+namespace Sonata\MediaBundle\Tests\DependencyInjection\Compiler;
 
 use PHPUnit\Framework\TestCase;
-use Sonata\MediaBundle\DependencyInjection\Compiler\ThumbnailCompilerPass;
```

Comment:
This should not be removed

Reply:
Why? This in under the same namespace.

Reply:
Oh, w8.

Reply:
I remove it before add `Tests` to namespace. Thanks for catching this.